.github/pr-commenter.yml

```diff
@@ -0,0 +1,361 @@
+        of new ones are proposed & discussed in [problem-specifications](https://github.com/exercism/problem-specifications) first, so that all tracks can potentially benefit.
+        Once a change is approved there by three maintainers, it becomes available for various language tracks to then implement.
+
+        <br>
+        <details>
+          <summary>‼️&nbsp;&nbsp;<b><em>Did You...</em></b></summary>
+        <br>
+        <table>
+        <td><br>
+
+         - [ ]  &nbsp;Update & rebase your branch with any (recent) upstream changes.
+         - [ ]  &nbsp;Spell and grammar check all prose changes.
+         - [ ]  &nbsp;Run [Prettier](https://prettier.io/) on all markdown and JSON files.
+             - (_Optionally_) run [yapf](https://github.com/google/yapf) ([_yapf config_](https://github.com/exercism/python/blob/main/.style.yapf)) to help format your code.
+         - [ ]  &nbsp;Run [flake8](http://flake8.pycqa.org/) with [_flake8 config_](https://github.com/exercism/python/blob/main/.flake8) to check general code style standards.
+         - [ ] &nbsp; Run [pylint](https://pylint.pycqa.org/en/v2.11.1/user_guide/index.html) with [_pylint config_](https://github.com/exercism/python/blob/main/pylintrc) to check extended code style standards.
+         - [ ]  &nbsp;Use pytest or the [python-track-test-runner](https://github.com/exercism/python-test-runner) to test any changed `example.py`/`exemplar.py`files
+         &nbsp;against their associated test files.
+         - [ ]  &nbsp;Similarly, use [pytest](https://docs.pytest.org/en/6.2.x/contents.html) or the [python-track-test-runner](https://github.com/exercism/python-test-runner) to test any changed _**test**_ files.
+             - Check that tests **fail** properly, as well as succeed.
+             &nbsp;(_**e.g.**, make some tests fail on purpose to "test the tests" & failure messages_).
+         - [ ]  &nbsp;Double-check all files for proper EOL.
+         - [ ]  &nbsp;[Regenerate](https://github.com/exercism/python/blob/main/CONTRIBUTING.md#generating-practice-exercise-documents) exercise documents when you modified or created a `hints.md` file for a practice exercise.
+         - [ ]  &nbsp;[Regenerate the test file](https://github.com/exercism/python/blob/main/CONTRIBUTING.md#auto-generated-test-files-and-test-templates) if you modified or created a `JinJa2` template file for a practice exercise.
+             -  Run the generated test file result against its `example.py`.
+         - [ ]  &nbsp;Run [`configlet-lint`](https://github.com/exercism/configlet#configlet-lint) if the track [config.json](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md), or any other exercise `config.json` has been modified.
+
+        <br>
+        </td>
+        </table>
+        <br>
+        </details>
+
+        <details>
+        <summary>✅️&nbsp;&nbsp;<b><em>Have You Checked...</em></b></summary>
+        <br>
+        <table>
+        <td>
+
+        .
```

Comment:
Is this period here purposefully?  (Unaware of what this does for yaml.)

Reply:
Yeah.  Sadly, its a fakeout for the GH markdown renderer.  It makes it so there is a line.  Otherwise, it would be swallowed.  A `<br>` there puts too much space in.

Reply:
Thanks.  I have seen it before, but never thought to ask.  It happened at least twice that I noticed here, so thought to ask.